libs/designsystem/accordion/src/accordion-item.component.spec.ts

```diff
@@ -22,54 +22,102 @@ describe('AccordionItemComponent', () => {
     });
   });
 
-  it('should create', () => {
-    expect(spectator.component).toBeTruthy();
-  });
+  describe('by default', () => {
+    it('should create', () => {
+      expect(spectator.component).toBeTruthy();
+    });
 
-  it('should have the configured title', () => {
-    const expectedText = 'Title';
-    expect(spectator.component.title).toEqual(expectedText);
-    expect(spectator.query('.title')).toHaveText(expectedText, true);
-  });
+    it('should have the configured title', () => {
+      const expectedText = 'Title';
+      expect(spectator.component.title).toEqual(expectedText);
+      expect(spectator.query('.title')).toHaveExactTrimmedText(expectedText);
+    });
 
-  it('should not be expanded', () => {
-    expect(spectator.component.isExpanded).toBeFalsy();
-  });
+    it('should have correct tabindex', () => {
+      expect(spectator.query('.header')).toHaveAttribute('tabIndex', '0');
+    });
+
+    it('should not be expanded', () => {
+      expect(spectator.component.isExpanded).toBeFalse();
+      expect(spectator.query('.header')).toHaveAttribute('aria-expanded', 'false');
+    });
+
+    it('should not be disabled', () => {
+      expect(spectator.component.isDisabled).toBeFalse();
+      expect(spectator.query('.header')).not.toHaveAttribute('aria-disabled');
+    });
 
-  it('should not show the content', () => {
-    expect(spectator.query('.content')).toHaveComputedStyle({
-      visibility: 'hidden',
+    it('should not show the content', () => {
+      expect(spectator.query('.content')).toBeHidden();
+    });
+
+    it('should emit the "toggle" event when expanded', () => {
+      spyOn(spectator.component.toggle, 'emit');
+      spectator.component.isDisabled = false;
+      spectator.component.isExpanded = false;
```

Comment:
`isDisabled` & `isExpanded` is set to their default values. Is it because other tests sets them to `true`?

Reply:
Nope, good catch, is redundant 👍🏻